release/src/required-checks.ts

```diff
@@ -0,0 +1,53 @@
+import "dotenv/config";
+import { Octokit } from "@octokit/rest";
+
+const { GITHUB_TOKEN, GITHUB_OWNER, GITHUB_REPO } = process.env;
+
+const github = new Octokit({ auth: GITHUB_TOKEN });
+
+const config = {
+  owner: GITHUB_OWNER || "",
+  repo: GITHUB_REPO || "",
+};
```

Comment:
@iethree we said on Slack to extract `getGithubCredentials` to utils and to then use it in this file, but when I started doing that, I realized the diff will be unnecessarily large. API will throw without owner and the repo anyway, so I think this is just way simpler, and it makes TypeScript happy.

Unless you strongly object, I propose we leave it like this in the first iteration.